test_version/version_specific_tests_dir/twitter_scrooge/dependency_version_in_filename.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+
+# this is used by a sh_test to test that a string is NOT
+# found in THE NAME OF a file. 
+# This is used to check if we are resolving the right version of scrooge_core
+
+set -x
+filename=$1
+dependency_version=$2
```

Comment:
where is this passed? I couldn't see it

Reply:
This is passed in the `--test-args` in `test_version.sh`

Reply:
Because rootpath returns two values? I thought it returned one and I only saw one arg in the sh_test

Reply:
Rootpath returns one value, the `filename`. The `dependency_version` depends on which test we're running, and we pass it via `test_args`, which inserts itself after the rootpath.

Reply:
However, I'll add a comment explaining that, I agree that it's very confusing.

Reply:
oh ok. I thought that you meant the args in the BUILD file (for some reason I ignored the test_version.sh reference).
Thanks.